index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="./style.css" />
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v6.0.0/css/all.css" integrity="sha384-3B6NwesSXE7YJlcLI9RpRqGf2p/EgVH8BgoKTaUrmKNDkHPStTQ3EyoYjCGXaOTS" crossorigin="anonymous">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+  <title>Portfolio</title>
+</head>
+
+<body>
+  <nav>
+    <img src="./images/logo.png" alt="my logo" class="logo" />
+    <ul class="navigation-container">
+      <li class=""><a href="#">Portfolio</a></li>
+      <li class=""><a href="#">About</a></li>
+      <li class=""><a href="#">Contact</a></li>
+    </ul>
+  </nav>
+  <div class="main-container">
+    <h1 class="main-element">I’m Olivier Rwubakwanayo</h1>
+    <h1 class="main-element">Glad to see you!</h1>
+    <p class="main-element">
+      I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+    </p>
+    <span class="main-element">LET’S CONNECT</span>
+    <ul class="main-element">
+      <li><i class="fa-brands fa-twitter"></i></li>
+      <li><i class="fa-brands fa-linkedin-in"></i></li>
+      <li><i class="fa-solid fa-m"></i></li>
+      <li><i class="fa-brands fa-github"></i></li>
+      <li><i class="fa-brands fa-angellist"></i></li>
```

Comment:
- [ ] Please add links to these icons, it will make more sense when the user will need to interact with the owner.
- [ ] Actually, you have not implemented the mobile version skeleton at all, please make sure you match this design here below

- [ YOUR DESIGN] 

<img width="315" alt="you" src="https://user-images.githubusercontent.com/79058364/158678938-3b3718df-42bc-4ff7-9578-95fe9da044cc.png">

- [ SAMPLE IN PROJECT REQUIREMENT ] 

<img width="708" alt="header" src="https://user-images.githubusercontent.com/79058364/158679262-7bb088a2-bc72-47f0-813d-9ce3c025073a.png">

- [ FIGMA ] 

<img width="299" alt="FIG" src="https://user-images.githubusercontent.com/79058364/158679714-ceb259bf-8de2-43ac-8f6e-5c0c44ea109a.png">

- [ ] Please use same `color, padding, margin, font-weight, font-size` as per figma design